lib/mixpanel-node.js

```diff
@@ -47,6 +49,8 @@ var create_client = function(token, config) {
             'verbose': metrics.config.verbose ? 1 : 0
         };
 
+        var protocol = metrics.config.protocol === 'http' ? http : https;
```

Comment:
I think it's better to throw if an unsupported protocol is given, instead of defaulting to anything.